crml/staking/src/tests.rs

```diff
@@ -2313,6 +2313,82 @@ fn invulnerables_are_not_slashed() {
 	});
 }
 
+#[test]
+fn invulnerables_can_be_set() {
```

Comment:
[Optional] Do we really need to add this test since it's very similar to `fn invulnerables_are_not_slashed` and `fn invulnerables_can_only_be_set_by_root` already covers it

Reply:
the other 2 new tests only check if storage is correct, but do not check for invulnerability.
this is sort of a "end to end" test that tests the whole thing. 
I don't mind removing it though